relay-dynamic-config/src/project.rs

```diff
@@ -48,6 +48,7 @@ pub struct ProjectConfig {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dynamic_sampling: Option<SamplingConfig>,
     /// Configuration for measurements.
+    /// NOTE: do not access directly, use DynamicMeasurementsConfig.
```

Comment:
Can't link to it directly without adding a `use crate.. dynamicmeasurementsconfig` statement. But when I do that, I get the linter error that the use statement is unused. any way to go around that?

Reply:
Does the linter complain if you fully qualify the import in the docstring? That is, instead of importing it with `use` and then linking it, adding the full path in the link directly.

Reply:
nice tip, thx!